README.md

```diff
@@ -266,7 +266,7 @@ The Acceptance test suite is located here: [acceptance-tests](./acceptance-tests
 When the ATH passes and there is a consensus that a release can be performed: 
 
 * Ensure that the person doing the release has permissions for all the blueocean modules here: https://github.com/jenkins-infra/repository-permissions-updater/tree/master/permissions (or it will fail)
-* Switch to the branch to release from (usually master)
+* Switch to the branch to release from (usually controller)
```

Comment:
the code use the canonical branch called `master` this has nothing to do with  a `controller` so it should be `main`

Reply:
> the code use the canonical branch called `master` this has nothing to do with  a `controller` so it should be `main`

Okay, will resolve that